tests/AbstractProcessTest.php

```diff
@@ -333,4 +406,16 @@ private function getPhpBinary()
 
         return $runtime->getBinary();
     }
+    
+    private function getPhpCommandLine($phpCode)
+    {
+    	$cmd = $this->getPhpBinary() . ' -r ' . escapeshellarg($phpCode);
```

Comment:
According to @auroraeosrose, `escapeshellarg()` is broken on Windows (which uses [different escaping](http://ss64.com/nt/syntax-esc.html). We should probably only conditionally call this for non-Windows systems.